gui-apps/foot/foot-1.10.3.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson xdg
+
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI="https://codeberg.org/dnkl/foot/archive/${PV}.tar.gz  -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN}"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://codeberg.org/dnkl/foot.git"
+fi
+
+DESCRIPTION="A fast, lightweight and minimalistic Wayland terminal emulator"
+HOMEPAGE="https://codeberg.org/dnkl/foot"
+LICENSE="MIT"
+SLOT="0"
+IUSE="ime +themes +grapheme-clustering"
+
+DEPEND="
+	dev-libs/libutf8proc:=
+	dev-libs/wayland
+	media-libs/fcft:=
+	media-libs/fontconfig
+	x11-libs/libxkbcommon
+	x11-libs/pixman
+"
+RDEPEND="
+	${DEPEND}
+	|| (
+		>=sys-libs/ncurses-6.3
+		~gui-apps/foot-terminfo-${PV}
+	)
+"
+BDEPEND="
+	app-text/scdoc
+	dev-libs/tllist
+	dev-libs/wayland-protocols
+	sys-libs/ncurses
```

Comment:
Is this really needed? I assume it was for running `tic` but it doesn't in this ebuild.